topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md

```diff
@@ -0,0 +1,759 @@
+
+> ### {% icon hands_on %} Hands-on: Labelling batches
+>
+> 1. {% tool [Manipulate AnnData](hhttps://humancellatlas.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.7.5+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: output of **Manipulate AnnData - Add new annotations** {% icon tool %}
+>    - *"Function to manipulate the object"*: `Rename categories of annotation`
+>    - *"Key for observations or variables annotation"*: `batch`
+>    - *"Comma-separated list of new categories"*: `N701,N702,N703,N704,N705,N706,N707`
+{: .hands_on}
+
+
+Huzzah! We are JUST about there. However, while we've been focussing on our cell metadata (sample, batch, genotype, etc.) to relabel the 'observations' in our object...
+
+# Mitochondrial reads
+
+Do you remember when we mentioned mitochondria early on in this tutorial? And how often in single cell samples, mitochondrial RNA is often an indicator of stress during dissociation? We should probably do something with our column of true/false in the gene annotation that tells us information about the cells. You will need to do this whether you have combined fastq files or are analysing just one (and thus skipping sections 4 & 5).
+
+> ### {% icon hands_on %} Hands-on: Calculating mitochondrial RNA in cells
+>
+> 1. {% tool [AnnData Operations](https://humancellatlas.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/anndata_ops/anndata_ops/0.0.3+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Input object in hdf5 AnnData format"*: output of **Manipulate AnnData - Rename categories** {% icon tool %}
+>    - *"Format of output object"*: `AnnData format`
+>    - *"Copy AnnData to .raw"*: `No`
+>    - *"Gene symbols field in AnnData"*: `NA.`
+>    - *"Flag genes that start with these names"*: `Insert Flag genes that start with these names`
+>    - *"Starts with"*: `True`
+>    - *"Var name"*: `mito`
+>    - *"Number of top genes"*: `50`
+{: .hands_on}
+
+{% icon congratulations %}Well done!  I strongly suggest have a play with the **Inspect AnnData** {% icon tool %} on your final `Pre-processed object` to see the wealth of information that has been added. You are now ready to move along to further filtering! There is a cheat that may save you time in the future though...
+
+# Pulling single cell data from public resources
+
+If you happen to be interested in analysing publicly available data, particularly from the [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/home), you may be interested in the following tool {% cite Moreno2020.04.08.032698 %} which rather skips forward all these steps in one! For this tutorial, the dataset can be seen [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6945/downloads) with experiment id of `E-MTAB-6945`.
```

Comment:
Just an FYI: we actually have .h5ad files on our FTP now that will make life much easier in future!

Reply:
Hot damn! When paired with the why-we-make-each-decision doc you guys are making, that is going to be a seriously powerful resource mate.

Reply:
[I mean the h5ad files, to be clear!]